dotCMS/src/main/java/com/dotcms/rest/api/v1/page/PageResource.java

```diff
@@ -1,44 +1,58 @@
 package com.dotcms.rest.api.v1.page;
 
-import java.util.Map;
-
-import javax.servlet.http.HttpServletRequest;
-
 import com.dotcms.repackage.com.google.common.annotations.VisibleForTesting;
-import com.dotcms.repackage.com.google.common.collect.Maps;
 import com.dotcms.repackage.javax.ws.rs.GET;
 import com.dotcms.repackage.javax.ws.rs.Path;
 import com.dotcms.repackage.javax.ws.rs.PathParam;
 import com.dotcms.repackage.javax.ws.rs.Produces;
 import com.dotcms.repackage.javax.ws.rs.core.Context;
 import com.dotcms.repackage.javax.ws.rs.core.MediaType;
-import com.dotcms.repackage.org.glassfish.jersey.server.JSONP;
+import com.dotcms.repackage.javax.ws.rs.core.Response;
+import com.dotcms.rest.InitDataObject;
 import com.dotcms.rest.WebResource;
 import com.dotcms.rest.annotation.NoCache;
+import com.dotcms.rest.exception.mapper.ExceptionMapperUtil;
 import com.dotmarketing.business.APILocator;
-import com.dotmarketing.portlets.contentlet.business.HostAPI;
+import com.dotmarketing.business.IdentifierAPI;
+import com.dotmarketing.business.VersionableAPI;
+import com.dotmarketing.business.web.HostWebAPI;
+import com.dotmarketing.business.web.WebAPILocator;
+import com.dotmarketing.exception.DotDataException;
+import com.dotmarketing.exception.DotSecurityException;
+import com.dotmarketing.portlets.containers.business.ContainerAPI;
 import com.dotmarketing.portlets.htmlpageasset.business.HTMLPageAssetAPI;
+import com.dotmarketing.portlets.languagesmanager.business.LanguageAPI;
+import com.dotmarketing.portlets.templates.business.TemplateAPI;
+import com.fasterxml.jackson.core.JsonProcessingException;
 import com.liferay.portal.model.User;
 
-@Path("/v1/pages")
-public class PageResource {
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
 
-    private final HTMLPageAssetAPI pageAPI;
-    private final HostAPI hostAPI;
+@Path("/v1/page")
+public class PageResource {
 
+    private final PageResourceHelper pageResourceHelper;
     private final WebResource webResource;
 
+    private final HostWebAPI hostWebAPI = WebAPILocator.getHostWebAPI();
+    private final IdentifierAPI identifierAPI = APILocator.getIdentifierAPI();
+    private final HTMLPageAssetAPI htmlPageAssetAPI = APILocator.getHTMLPageAssetAPI();
+    private final LanguageAPI languageAPI = APILocator.getLanguageAPI();
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove this unused "languageAPI" private field. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1068)